<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDialog</class>
 <widget class="QDialog" name="NewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="lblIcon">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpRunNew">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Run new application</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblRunNewApplication">
        <property name="text">
         <string>Application:</string>
        </property>
        <property name="buddy">
         <cstring>lineRunNewApplication</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="lblRunNewFolder">
        <property name="text">
         <string>Folder:</string>
        </property>
        <property name="buddy">
         <cstring>lineRunNewFolder</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="lblRunNewArguments">
        <property name="text">
         <string>Arguments:</string>
        </property>
        <property name="buddy">
         <cstring>lineRunNewArguments</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineRunNewArguments"/>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="btnRunNewFolder">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineRunNewApplication"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineRunNewFolder"/>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="btnRunNewApplication">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="btnRunNew">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpAttach">
     <property name="title">
      <string>Attach to running process</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="treeAttach">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>PID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CPU usage %</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Process</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="btnAttach"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpOptions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
      <item row="0" column="1">
       <widget class="QCheckBox" name="chkDownloadSymbols">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblDownloadSymbols">
        <property name="text">
         <string>Download Microsoft symbols:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QSpinBox" name="spnOptionsSamplingFreq">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblOptionsSamplingFreq">
        <property name="text">
         <string>Sampling frequency (ms):</string>
        </property>
        <property name="buddy">
         <cstring>spnOptionsSamplingFreq</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblDownloadInfo">
        <property name="text">
         <string>&lt;a href=&quot;?&quot;&gt;(?)&lt;/a&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineRunNewApplication</tabstop>
  <tabstop>btnRunNewApplication</tabstop>
  <tabstop>lineRunNewFolder</tabstop>
  <tabstop>btnRunNewFolder</tabstop>
  <tabstop>lineRunNewArguments</tabstop>
  <tabstop>btnRunNew</tabstop>
  <tabstop>treeAttach</tabstop>
  <tabstop>spnOptionsSamplingFreq</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>accepted()</signal>
   <receiver>NewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>rejected()</signal>
   <receiver>NewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
